apps/web-giddh/src/app/subscription/subscription.component.ts

```diff
@@ -1,769 +1,372 @@
-import { SubscriptionComponentStore } from './utility/subscription.store';
-import { MatTableDataSource } from '@angular/material/table';
-import { MatPaginator } from '@angular/material/paginator';
-import { API_COUNT_LIMIT, PAGE_SIZE_OPTIONS } from '../app.constant';
-import { FormBuilder, FormGroup } from '@angular/forms';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
+import { Observable, ReplaySubject, takeUntil } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
 import { AppState } from '../store';
-import { Store } from '@ngrx/store';
-import { SubscriptionsUser } from '../models/api-models/Subscriptions';
-import { MatMenuTrigger } from '@angular/material/menu';
-import { CompanyListDialogComponent } from './company-list-dialog/company-list-dialog.component';
-import { TransferDialogComponent } from './transfer-dialog/transfer-dialog.component';
+import { select, Store } from '@ngrx/store';
 import { GeneralActions } from '../actions/general/general.actions';
-import { BuyPlanComponentStore } from './buy-plan/utility/buy-plan.store';
 import { ToasterService } from '../services/toaster.service';
+import { CompanyResponse } from '../models/api-models/Company';
+import { SignupWithMobile, UserDetails, VerifyMobileModel } from '../models/api-models/loginModels';
+import { GIDDH_DATE_FORMAT_DD_MM_YYYY, GIDDH_DATE_FORMAT_UI } from '../shared/helpers/defaultDateFormat';
+import { BsModalRef } from 'ngx-bootstrap/modal';
+import { BreakpointObserver } from '@angular/cdk/layout';
+import { ClipboardService } from 'ngx-clipboard';
+import { LoginActions } from '../actions/login.action';
+import { SessionActions } from '../actions/session.action';
+import { API_POSTMAN_DOC_URL, BootstrapToggleSwitch } from '../app.constant';
+import { cloneDeep } from '../lodash-optimized';
+import { AuthenticationService } from '../services/authentication.service';
+import * as dayjs from 'dayjs';
+import * as duration from 'dayjs/plugin/duration';
+dayjs.extend(duration)
 @Component({
-    selector: 'subscription',
+    selector: 'app-subscription',
     templateUrl: './subscription.component.html',
     styleUrls: ['./subscription.component.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush,
-    providers: [SubscriptionComponentStore, BuyPlanComponentStore]
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SubscriptionComponent implements OnInit, OnDestroy {
-    /** Mat menu instance reference */
-    @ViewChild(MatMenuTrigger) menu: MatMenuTrigger;
-    /** Holds Paginator Reference */
-    @ViewChild(MatPaginator) paginator!: MatPaginator;
-    /** This will use for move company in to another company  */
-    @ViewChild("moveCompany", { static: false }) public moveCompany: any;
+    /** True If Auth key copied and used toggle Copy text */
+    public isCopied: boolean = false;
+    public userAuthKey: string = '';
```

Comment:
Write js doc  and variable type if required

Reply:
This is an old module